.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## Overview
+**TL;DR**
+<One to two sentence description of the issue you are encountering or trying to solve.>
+
+<Link to related issue. Type `closes #RELATEDISSUENUMBER` to establish a link.>
+
+### Questions
+<If relevant, write a list of questions that you'd like to discuss related to the changes that you have made.>
+
+### Next Steps
+<If incomplete, create a task list of items that are still being worked on within the Pull Request.>
+
```

Comment:
I think we run into the same problem here, that this can't be done without collaborator access.